source/blog/2017-01-03-ovirt-system-tests-to-the-rescue.html.md

```diff
@@ -36,18 +36,32 @@ I know you can't wait to hear about how to run OST on your patch, so I'll just j
     and a new build will be triggered on the project 'build-on-demand' jobs.<br>
     Once the 'build-on-demand' job finished building, write down the job URL,
-    for e.g: [vdsm-master-build-on-demand](http://jenkins.ovirt.org/job/vdsm_master_build-artifacts-on-demand-el7-x86_64/9/).
+    for e.g: [vdsm-master-build-on-demand](http://jenkins.ovirt.org/job/vdsm_master_build-artifacts-on-demand-el7-x86_64//).
     (btw, you can do this for as many oVirt projects you want and have a list of URLs).
 
 
 * *Run the manual OST job with your custom RPMs*
 
-    Now that you have your custom RPMs ready, your JUST a click away from running OST
+    Now that you have your custom RPMs ready, you're JUST a few clicks away from running OST
     on them.<br>
     * Login to [Jenkins](http://jenkins.ovirt.org) (make sure you have 'dev role' permissions, if not open a ticket to infra)<br>
-    * Go to the [OST Manual job](http://jenkins.ovirt.org/job/ovirt_master_system-tests_manual/) for your relevant version (usually master).<br>
+    * Go to the [OST Manual job](http://jenkins.ovirt.org/job/ovirt-system-tests_manual/).<br>
     * Click on 'build with parameters' menu ( on the left side )
-    * Now add all the URLs you have with the custom RPMs ( one per line ),for e.g [vdsm-build](http://jenkins.ovirt.org/job/vdsm_master_build-artifacts-on-demand-el7-x86_64/9/)<br>
+    * Choose the oVirt version: This is the version that we actually tested.<br>
+      (in the upgrade suites, this is the version that we will be upgrading to)<br>
+    * Choose the suite type you want to run:<br>
+        <u>basic:</u> Run engine-setup, and basic tests (bootstrap, sanity and etc)<br>
+        <u>upgrade:</u> Initialize the engine with a base version, test if an upgrade to the target version is possible.<br>
+        Here we have 3 options:<br>
+        upgrade-from-rc: The base version installed (before the upgrade) is the current release candidate<br>
+        upgrade-from-release: Depends on the target version, the current official release will be set as the base installed version from which we will upgrade. e.g, if you choose upgrade from release for oVirt-4.1, the suite will install the official release of <b>4.1</b> and upgrade to the latest repo with your patch on top of it.<br>
+        upgrade-from-prevrelease: Depends on the target version, the previous official release will be set as the base installed version from which we will upgrade. e.g, if you choose upgrade from release for oVirt-4.1, the suite will install the official release of <b>4.0</b> and upgrade to the latest repo with your patch on top of it.<br>
+    * Add all the URLs you have with the custom RPMs ( one per line ),for e.g [vdsm-build](http://jenkins.ovirt.org/job/vdsm_master_build-artifacts-on-demand-el7-x86_64/lastSuccessfulBuild/)<br>
+    * Choose a fallback_repo:<br>
+        A base repo that will be used 'under' your tested patch.<br>
```

Comment:
Suggested edits
Add indented bullet
What do you mean by 'under'? It is always best to avoid these types of quotation marks.